base/applications/drwtsn32/lang/it-IT.rc

```diff
@@ -0,0 +1,14 @@
+/*
+ * PROJECT:     Dr. Watson crash reporter
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Italian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_APP_TITLE "ReactOS Crash Reporter"
+    IDS_USER_ALERT_MESSAGE "L'applicazione %ls ha smesso di funzionare. L'informazione su questo crash è stato salvato su:\r\n\r\n%ls"
```

Comment:
I have a small correction
Line 13: 
`IDS_USER_ALERT_MESSAGE "L'applicazione %ls ha smesso di funzionare. Le informazioni su questo crash sono state salvate in:\r\n\r\n%ls"`

Reply:
Changes committed.

Grazie. :)